tests/remote_attestation/maa/sgxlkl_enclave/oe_private/oe_defs.h

```diff
@@ -0,0 +1,150 @@
+#ifndef OE_DEFS_H
```

Comment:
This file should go away if we expose the cert-related APIs through OE host verify.